hadoop-ozone/s3gateway/src/test/java/org/apache/hadoop/ozone/s3/endpoint/TestPermissionCheck.java

```diff
@@ -266,9 +274,10 @@ public void testGetKey() throws IOException {
 
   @Test
   public void testPutKey() throws IOException {
```

Comment:
Nit: Not part of this review but it would be good to actually describe the intent of this test.

Reply:
And others in this file.

Reply:
Thanks for the detailed review, @kerneltime.
The class name already describes the context of the test-cases so all its methods can implicitly inherit. Also, I just added a description to the class javadoc to describe the test intent.